ui/tests/acceptance/auth-test.js

```diff
@@ -55,53 +41,102 @@ module('Acceptance | auth', function (hooks) {
     assert.strictEqual(component.tokenValue, '', 'it clears the token value when toggling methods');
   });
 
-  test('it sends the right attributes when authenticating', async function (assert) {
-    assert.expect(8);
-    const backends = supportedAuthBackends();
-    await visit('/vault/auth');
-    for (const backend of backends.reverse()) {
-      await component.selectMethod(backend.type);
-      if (backend.type === 'github') {
-        await component.token('token');
-      }
-      if (backend.type === 'jwt' || backend.type === 'oidc') {
-        await jwtComponent.role('test');
-      }
-      await component.login();
-      const lastRequest = this.server.passthroughRequests[this.server.passthroughRequests.length - 1];
-      const body = JSON.parse(lastRequest.requestBody);
+  module('it sends the right attributes when authenticating', function (hooks) {
+    hooks.beforeEach(function () {
+      this.assertReq = () => {};
+      this.server.get('/auth/token/lookup-self', (schema, req) => {
+        this.assertReq(req);
+        req.passthrough();
+      });
+      this.server.post('/auth/github/login', (schema, req) => {
+        // This one is for github only
+        this.assertReq(req);
+        req.passthrough();
+      });
+      this.server.post('/auth/:mount/oidc/auth_url', (schema, req) => {
+        // For JWT and OIDC
+        this.assertReq(req);
+        req.passthrough();
+      });
+      this.server.post('/auth/:mount/login/:username', (schema, req) => {
+        this.assertReq(req);
+        req.passthrough();
+      });
+      this.expected = {
```

Comment:
This is a longer but more explicit way of checking that the url and data are doing what we expect on login